lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
@kares I agree putting PG as non-global is best if it works.  I respect what you say but I also think we need to work with the idea we can change things upstream (AR) to make things easier since there clearly is not enough contributions to keep this up to date.  So I am probably too optimistic to thing we can affect some change and reduce or eliminate this?